src/main/java/marquez/api/ListNamespacesResponse.java

```diff
@@ -8,13 +8,12 @@
 import marquez.core.models.Namespace;
 
 public class ListNamespacesResponse {
-  // TODO: Remove use of CoreNamespaceToApiNamespaceMapper
+
   CoreNamespaceToApiNamespaceMapper namespaceMapper = new CoreNamespaceToApiNamespaceMapper();
   private final List<Namespace> namespaces;
 
-  // TODO: Constructor should accept marquez.api.models.Namespace instead
-  public ListNamespacesResponse(@JsonProperty("namespaces") List<Namespace> namespaces) {
-    this.namespaces = namespaces;
+  public ListNamespacesResponse(@JsonProperty("namespaces") List<Namespace> namespaceList) {
+    this.namespaces = namespaceList;
   }
 
   @JsonProperty("namespaces")
```

Comment:
Think this pulling changes from your branch that are out dated? we'll want to add back the comments + rename (see https://github.com/MarquezProject/marquez/blob/issue-100/src/main/java/marquez/api/ListNamespacesResponse.java)

Reply:
Good catch -- I remember picking this one in the rebase over the newer one. I'll fix this.

Reply:
Woops, yeah looks like I picked the wrong patch during the rebase, will fix.

Reply:
reverted this file to the latest from `issue-100`.

Reply:
👍